### PR TITLE
Fix outdated kubernetes documentation for customConfigMap

### PR DESCRIPTION
**Pull request by johanhelsing-attensi**
It is now named existingConfigMap

I just ran:

git grep -l 'customConfigMap' | xargs sed -i 's/customConfigMap/existingConfigMap/g'

#361

### CODE REVIEW

File: website/translated_docs/ar/kubernetes.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ kubectl create configmap verdaccio-config --from-file ./config.yaml
 Now you can deploy the Verdaccio Helm chart and specify which configuration to use:
 
 ```bash
```
Comment (juanpicado) [inline website/translated_docs/ar/kubernetes.md:88]: Thanks for the big effort, but actually only `docs/kubernetes.md` is need it to be changed :) details are on the CONTRIBUTING guidelines, other files are automated.

**Review by juanpicado — CHANGES_REQUESTED** [review]
Please undo changes https://github.com/verdaccio/website/blob/master/CONTRIBUTING.md#source-code-website

File: website/translated_docs/ar/kubernetes.md (hunk 1 above)
Comment (johanhelsing-attensi) [reply website/translated_docs/ar/kubernetes.md]: Thanks, that's a relief, wish I'd known before, then I'd probably made a pr without being asked.

Force-pushed without the translations.

**Review by juanpicado — APPROVED** [review]
:) thanks @johanhelsing-attensi

**Pull request merged**